argparse/argparse_cwl_translation.py

```diff
@@ -125,4 +125,4 @@ def get_cwl_type(py_type):
         elif type(py_type).__name__ == 'builtin_function_or_method' or type(py_type).__name__ == 'FileType':
             return 'File'
         else:  # type given as a string: 'str', 'int' etc.
-            return PY_TO_CWL_TYPES[py_type]
+            return PY_TO_CWL_TYPES.get(py_type,None)
```

Comment:
@mr-c https://github.com/erasche/argparse2tool/pull/38 fails for the same reason, so the problem is not in the code, but in checks themselves. #41 will fix that